lib/web_apis/api_importer.es6.js

```diff
@@ -71,6 +73,22 @@ foam.CLASS({
         });
       },
     },
+    {
+      class: 'Boolean',
+      name: 'toCloudDatastore',
+      documentation: `An optional boolean property. When set true,
+        the importer will import webInterface and browserAPIs to
+        DAO with a rate limit.`,
+      values: false,
+    },
+    {
+      name: 'limiter',
+      documentation: `An RateLimiter object to constraint the number
```

Comment:
nits: `A RateLimiter` and `to constrain`. Suggested reword: `A RateLimiter responsible for limiting the rate of put()s to DatastoreDAO (only applicable when toCloudDatastore = true).`.